docs/src/pages/components/buttons/UnstyledButtonCustom.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import ButtonUnstyled, {
+  ButtonUnstyledProps,
+  buttonUnstyledClasses,
+} from '@material-ui/unstyled/ButtonUnstyled';
+import StarOutlineIcon from '@material-ui/icons/StarOutline';
+import { styled } from '@material-ui/system';
+
+const ButtonRoot = React.forwardRef(
+  (props: React.PropsWithChildren<{}>, ref: React.ForwardedRef<any>) => {
```

Comment:
Always name these to get a name in warnings/devtools via `React.forwardRef(function Named() {})`